tools/gatk_selectvariants.cwl

```diff
@@ -10,28 +10,25 @@ requirements:
   - class: ResourceRequirement
     ramMin: 8000
     coresMin: 4
-baseCommand: ["/bin/bash", "-c"]
-arguments:
-  - position: 0
-    shellQuote: false
-    valueFrom: >-
-      set -eo pipefail
+  - class: InitialWorkDirRequirement
+    listing:
+      - entryname: select_pass.sh
+        entry: |
+          #!/usr/bin/env bash
 
-      ${
-        var run_mode = inputs.mode;
-        if (run_mode == 'grep' || run_mode == 'gatk'){
-          var in_vcf = inputs.input_vcf.path;
-          var out_vcf = inputs.output_basename + '.' + inputs.tool_name + '.PASS.vcf.gz';
-          var cmd = '/gatk SelectVariants --java-options "-Xmx8000m" -V ' + in_vcf +  ' -O ' + out_vcf + ' --exclude-filtered TRUE';
-          if (run_mode == 'grep'){
-            cmd = 'zcat ' + in_vcf + ' | grep -E "^#|PASS" | bgzip > ' + out_vcf + '; tabix ' + out_vcf;
-          }
-          return cmd;
-        }
-        else{
-          throw new Error(run_mode + ' is a not a valid mode.  Choices are gatk or grep.');
-        }
-      }
+          set -eo pipefail
+
+          if [[ $(inputs.mode) = gatk ]]
```

Comment:
this is too clean and easy to read. stop it.

Reply:
no